contract-ui/tabs/overview/components/MarketplaceDetails.tsx

```diff
@@ -0,0 +1,366 @@
+              label="view_all_listings"
+              color="blue.400"
+              _light={{
+                color: "blue.600",
+              }}
+              gap={4}
+              href={listingsHref}
+            >
+              View listings -&gt;
+            </TrackedLink>
+          </Flex>
+          <ListingCards
+            listings={listings}
+            isLoading={listingsQuery.isLoading}
+            trackingCategory={trackingCategory}
+          />
+        </>
+      )}
+    </Flex>
+  );
+};
+
+const MarketplaceV3Details: React.FC<
+  MarketplaceDetailsVersionProps<MarketplaceV3>
+> = ({ contract, trackingCategory }) => {
+  const directListingsHref = useTabHref("direct-listings");
+  const englishAuctionsHref = useTabHref("english-auctions");
+
+  const directListingsCountQuery = useDirectListingsCount(contract);
+  const englishAuctionsCountQuery = useEnglishAuctionsCount(contract);
+
+  const directListingsQuery = useDirectListings(contract, {
+    count: 3,
+    start: BigNumber.from(directListingsCountQuery?.data || 3)?.toNumber() - 3,
+  });
+  const englishAuctionsQuery = useEnglishAuctions(contract, {
+    count: 3,
+    start: BigNumber.from(englishAuctionsCountQuery?.data || 3)?.toNumber() - 3,
+  });
+
+  const directListings = useMemo(
+    () =>
+      directListingsQuery?.data
+        ?.map<ListingData>((v) => ({
+          ...v,
+          sellerAddress: v.creatorAddress,
+          type: "direct-listing",
+          currencyValue: v.currencyValuePerToken,
+        }))
+        .reverse() || [],
+    [directListingsQuery?.data],
+  );
+
+  const englishAuctions = useMemo(
+    () =>
+      englishAuctionsQuery?.data
+        ?.map<ListingData>((v) => ({
+          ...v,
+          sellerAddress: v.creatorAddress,
+          type: "english-auction",
+          currencyValue: v.buyoutCurrencyValue,
+        }))
+        .reverse() || [],
+    [englishAuctionsQuery?.data],
+  );
+
+  return (
+    <Flex gap={6} flexDirection="column">
+      <Heading size="title.sm">Listings</Heading>
+      <ListingStatsV3 contract={contract} />
+      {!directListingsQuery.isLoading && directListings.length === 0 ? null : (
+        <>
+          <Flex align="center" justify="space-between" w="full">
+            <Heading size="label.lg">Direct Listings</Heading>
+            <TrackedLink
+              category={trackingCategory}
+              label="view_all_direct_listings"
+              color="blue.400"
+              _light={{
+                color: "blue.600",
+              }}
+              gap={4}
+              href={directListingsHref}
+            >
+              View direct listings -&gt;
+            </TrackedLink>
+          </Flex>
+          <ListingCards
+            listings={directListings}
+            isLoading={directListingsQuery.isLoading}
+            trackingCategory={trackingCategory}
+            isMarketplaceV3
+          />
+        </>
+      )}
+      {!englishAuctionsQuery.isLoading &&
```

Comment:
not sure why since I don't see anything weird at first glance with the code but english auctions wasn't loading for me on my testing

Reply:
here's the url of my marketplace to see it: https://thirdweb-www-git-sid-contract-overview-token.thirdweb-preview.com/mumbai/0xCd3561F21E43f285c0f3ae1DA08d5C0BC1254660